### PR TITLE
drop highest_status from addon

### PR DESCRIPTION
**Pull request by eviljeff**

### CODE REVIEW

File: src/olympia/amo/tasks.py [hunk 1]
```diff
@@ -106,18 +105,6 @@ def delete_anonymous_collections(items, **kw):
 
 
 @task
-def delete_incomplete_addons(items, **kw):
```
Comment (eviljeff) [inline src/olympia/amo/tasks.py]: I couldn't see it used anywhere

Comment (EnTeQuAk) [inline src/olympia/amo/tasks.py]: good catch :+1:

**Comment by EnTeQuAk** [comment]
I can't say much about wether it's useful or not but it seems tor remove cruft, so r+

**Comment by eviljeff** [comment]
Note I've remove the migration as it would cause errors during the deployment window.  This will have to run in the following push.

**Pull request merged**